### DISCUSSION TITLE
flashin pre configured klipper for btt pi v1.2

### DISCUSSION THREAD

**Discussion by t4k3sh1** [bigtreetech/CB1#184] [Q&A]
hi, im new in this world, how i flash the pre configured file on the btt pi? like a normal os with raspberry etcher using an microsd card?

**Comment by frutabruta** [comment]
Etcher should probably work, but the recommended program is Raspberry Pi imager. It can even accept images in an archive (.zip etc.)
But be aware that the WiFi, password, keyboard language and others CANNOT be set directly in the Raspberry Pi imager.

**Comment by Ireq007** [comment]
Hi, I'm new to the world of Vorona. I have a BTT Manta M8P+CB1 motherboard. I'm trying to flash cb1_debian 12 version 3.0.0 to a microSD card. However, after flashing and checking with the rosbperry imager, I get an error and it tries to re-flash the file. Any advice? I'm using Windows 11.
